extensions/wikia/Answers/SpecialCreateDefaultQuestionPage.php

```diff
@@ -1,58 +1,84 @@
 <?php
 class CreateQuestionPage extends UnlistedSpecialPage {
 
+	const QS_KEY_QUESTION = 'questiontitle';
+
 	function __construct(){
 		parent::__construct("CreateQuestionPage");
 	}
 
-	function execute( $question ){
-		global $wgRequest, $wgOut, $wgUser;
+	function execute( $question ) {
+		if ( $this->getRequest()->wasPosted() ) {
+			$this->createQuestion();
+			return;
+		}
 
-                if ( wfReadOnly() ) {
-                        return false;
-                }
+		if ( !empty( $question ) ) {
+			$this->redirectToHelpPage();
+		}
+	}
 
-                //don't allow blocked users to ask questions, duh
-                if( $wgUser->isBlocked() ){
-					throw new UserBlockedError( $wgUser->getBlock() );
-                }
+	private function createQuestion() {
+		if ( !$this->getRequest()->wasPosted()
```

Comment:
We've already checked this on L10 so this will always be `false`.  Can just remove it.